### ISSUE TITLE
Collection sheet submits in offline mode in Individual collection sheet

### ISSUE THREAD

**Issue by iamsh4shank** [openMF/android-client#1503]
**Summary:** 

If we enable the offline mode then also the collection sheet is submitted in individual collection sheet.

**Steps to reproduce:** 

Click on individual collection sheet> Fill the fields> click on generate> click on fill now>then press submit 

**Expected behaviour:** 

On offline mode it should show an error message or a toaster for notifying that the user is in offline mode.

**Observed behaviour:** 

If we enable the offline mode then also the collection sheet is submitted in individual collection sheet.

**Device and Android version:** 

Android 9.0
Realme 5

**Screenshots:** 

![ezgif com-resize](https://user-images.githubusercontent.com/52889867/79640447-4e3b7200-81af-11ea-9eac-880333e583da.gif)

**Comment by robustTechie** [comment]
I would like to work on this.

**Comment by Saral-code** [comment]
I would like to work on this.

**Comment by mananvirmani611** [comment]
@robustTechie @Saral-code are you presently working on this issue ??

**Comment by robustTechie** [comment]
@Saral-code you can work on this

**Comment by Saral-code** [comment]
> @robustTechie @Saral-code are you presently working on this issue ??

No, I am not.

**Comment by therajanmaurya** [comment]
No, we have offline mode that gives user to see synced data but It does not stop user to use any other feature that is accessible
